.github/workflows/feature.yml

```diff
@@ -0,0 +1,82 @@
+name: Build and deploy feature
+
+on:
+  push:
+    branches:
+      - 'feature/*'
+
+jobs:
+  build:
+    name: build
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@v2
+
+      - name: Check out the repo
+        uses: actions/checkout@v2
+
+      - name: Git vars
+        id: vars
+        shell: bash
+        run: |
+          echo "::set-output name=sha_short::$(git rev-parse --short HEAD)"
+
+      - name: Login to Docker Hub
+        run: docker login -u ${{ secrets.DOCKERHUB_USERNAME }} -p ${{ secrets.DOCKERHUB_TOKEN }}
+
+      - name: Build image
+        run: |
+          make CHAINNET=${{ steps.vars.outputs.sha_short }} BINARY=sifnoded build-image
```

Comment:
We might need to handle this differently with the work I am in the process of doing so we can create a testnet build with testnet code ie fauct. Let talk when you are online to work this out.